components/Chips/src/MDCChipView.h

```diff
@@ -118,6 +118,17 @@
  */
 @property(nonatomic, assign) BOOL enableRippleBehavior;
 
+/**
+ Set this flag to YES/true to allow the chip to be in the selected state when tapped.
+ When selection is allowed, tapping a chip automatically (after a momentary ripple animation)
+ toggles the chip's selected state, applying all custom theming related to the selected state (or
```

Comment:
Please remove the documentation on theming here; it’s implied in all of our components that they will respect their style properties in various states.